### DISCUSSION TITLE
Unable to install qc-selector package using "pip install qc-selector"

### DISCUSSION THREAD

**Discussion by Shashankss1205** [theochem/Selector#192] [Q&A] [answered]
![image](https://github.com/theochem/Selector/assets/123410790/ae67bcc9-2c4a-47c1-81aa-455f5f47ef12)

**Comment by PaulWAyers** [comment, marked as answer]
@FanwangM can you update the installation instructions? I think that the pip and conda releases are not out yet.

So @Shashankss1205 you need to install from source, `pip install git+https://github.com/theochem/Selector.git` should work.

**Reply by Shashankss1205** [reply]
Thank You Sir for your prompt response!!
I will proceed onto with the suggested approach.

**Reply by FanwangM** [reply]
Issue addressed in #193. Thanks for pointing it out. @Shashankss1205

**Reply by Shashankss1205** [reply]
My pleasure. @FanwangM
